### PR TITLE
added query endpoint 

### PR DESCRIPTION
**Pull request by heron2014**
Related to [issue 166](https://github.com/FAC-GM/app/issues/166) and [issue 177](https://github.com/FAC-GM/app/issues/177)

@nelsonic can you please check this PR?

### CODE REVIEW

File: test/query.test.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+});
+
+describe('return a search results, when user is authenticated', function () {
+  it('returns specific search results', function (done) {
+            
+    Server.init(0, function (err, server) {
+
+      expect(err).to.not.exist();
+      server.inject({url: '/query/1?job=javascript&fullname=Anita&location=London', headers: { cookie: "token=" + token }}, function (res) {
+        expect(res.statusCode).to.equal(200);
```
Comment (nelsonic) [inline test/query.test.js]: What *results* are you expecting...?

Comment (SimonLab) [inline test/query.test.js]: It's true that our test might be a bit weak. It should be better now

**Comment by nelsonic** [comment]
@SimonLab did you end up creating the *real data* fixture on Heroku?

**Comment by SimonLab** [comment]
@nelsonic It's in progress but we need a lot more time for that as we need to change the way we are running all our current tests

**Comment by nelsonic** [comment]
Ok. So it will be done as part of: https://github.com/FAC-GM/app/issues/62 ?

**Comment by SimonLab** [comment]
Yes that's right.

**Comment by nelsonic** [comment]
Ok. cool. lets merge this now then. :+1:

**Pull request merged**